From 7dd55a2f07eacb197f83306199022bd52e5a2f51 Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Wed, 23 Mar 2022 18:30:39 +0200
Subject: [PATCH 457/471] ARM: configs: sama7: enable WM8731 codec driver

The WM8731 codec driver is needed when the Mikroe Proto Codec board
sound card driver is replaced by another sound card (such as Microchip's
ASRC card) and the WM8731 codec is used.

Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
---
 arch/arm/configs/sama7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/sama7_defconfig b/arch/arm/configs/sama7_defconfig
index 386efc705193..6a75c59cd93d 100644
--- a/arch/arm/configs/sama7_defconfig
+++ b/arch/arm/configs/sama7_defconfig
@@ -163,6 +163,7 @@ CONFIG_SND_MCHP_SOC_ASRC=y
 CONFIG_SND_SOC_DMIC=y
 CONFIG_SND_SOC_PCM5102A=y
 CONFIG_SND_SOC_SPDIF=y
+CONFIG_SND_SOC_WM8731=m
 CONFIG_SND_SIMPLE_CARD=y
 CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
-- 
2.34.1

